x-pack/plugins/secops/public/components/ip/index.test.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import toJson from 'enzyme-to-json';
+import * as React from 'react';
+import { mountWithIntl, shallowWithIntl } from 'test_utils/enzyme_helpers';
+import { TestProviders } from 'x-pack/plugins/secops/public/mock/test_providers';
```

Comment:
You want to use relative paths for things like `TestProviders` and not `x-pack/...`